tests/test_ui/test_account_page.py

```diff
@@ -22,11 +25,45 @@ def test_account_creation(edit_default_admin_values, register_user, login, user_
 @pytest.mark.usefixtures('driver')
 def test_additional_account_creation(register_user, login, user_scenario):
     account_page_steps.navigate_to_open_new_account()
-    account_page_assert_steps.check_accounts_overview_navigation(test_messages["open_account_title"])
+    account_page_assert_steps.check_page_navigation(test_messages["open_account_title"])
     account_page_assert_steps.check_minimal_amount_message(DEFAULT_MIN_BALANCE)
     account_page_steps.create_savings_account()
     account_page_assert_steps.check_opened_account_title(test_messages["account_opened"])
     account_page_steps.navigate_to_accounts_overview()
     account_page_assert_steps.check_first_account_balance(DEFAULT_BALANCE, DEFAULT_MIN_BALANCE)
     account_page_assert_steps.check_second_account_balance(DEFAULT_MIN_BALANCE)
 
+
+@allure.description('ACU-3')
+@allure.title('User can transfer money between accounts')
+@allure.tag('Account Page')
+@pytest.mark.usefixtures('driver')
+@pytest.mark.parametrize('transfer_amount', [10, 0, -10, 100.5])
+def test_transfer_money_between_accounts(register_user, login, user_scenario, transfer_amount):
+    from_account_id = \
+     get_text_from_json(user_api_steps.get_customer_accounts_info(FIRSTLY_CREATED_USER_CUSTOMER_ID))[0]["id"]
+    new_account_id = get_text_from_json(
+        user_api_steps.post_account_for_existing_user(FIRSTLY_CREATED_USER_CUSTOMER_ID, 1, from_account_id))["id"]
```

Comment:
1 - is a magic number here. You should describe acc types somewhere

Reply:
Did's understand logic. You've got FIRSTLY_CREATED_USER_CUSTOMER_ID systemId and send some amount to the same account but based on some other id?